### PR TITLE
[Vis Builder] Rename wizard to visBuilder in class name, type name and function name

### PR DESCRIPTION
**Pull request by abbyhu2000**
Reame wizard to visBuilder

Signed-off-by: abbyhu2000 <abigailhu2000@gmail.com>

### Description
Rename wizard to visBuilder in class name, type name and function name
 
### Issues Resolved
resolves part of #1706 
 
### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/vis_builder/common/vis_builder_saved_object_attributes.ts [hunk 1]
```diff
@@ -5,9 +5,9 @@
 
 import { SavedObjectAttributes } from '../../../core/types';
 
-export const WIZARD_SAVED_OBJECT = 'wizard';
+export const VISBUILDER_SAVED_OBJECT = 'wizard';
```
Comment (kavilla) [inline src/plugins/vis_builder/common/vis_builder_saved_object_attributes.ts:8]: `VIS_BUILDER`*?

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 * [Vis Builder] Change wizard to vis_builder in file names and paths ([#2587](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2587))
 * [Multi DataSource] Address UX comments on Data source list and create page ([#2625](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2625))
 * [Vis Builder] Rename wizard to visBuilder in i18n id and formatted message id ([#2635](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2635))
+* [Vis Builder] Rename wizard to visBuilder in class name, type name and function name ([#2639](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2639))
```
Comment (ashwin-pc) [inline CHANGELOG.md:29]: We may need to consolidate the changelog for all the renaming stuff. but not for this PR though.

**Review by ashwin-pc — APPROVED** [review]

**Review by ananzh — APPROVED** [review]

File: src/plugins/vis_builder/common/vis_builder_saved_object_attributes.ts (hunk 1 above)
Comment (abbyhu2000) [reply src/plugins/vis_builder/common/vis_builder_saved_object_attributes.ts:8]: will change in next PR

**Comment by AMoo-Miki** [comment]
Will try to backport from the 2.x PR.

**Pull request merged**